.github/workflows/backend-cd-prod.yml

```
@@ -0,0 +1,113 @@
+          DEPLOY_JAR_PATH="${DEPLOY_DIRECTORY}/${BUILD_JAR_NAME}"
+          echo "BUILD_JAR_PATH=${BUILD_JAR_PATH}"
+          echo "BUILD_JAR_NAME=${BUILD_JAR_NAME}"
+          echo "DEPLOY_JAR_PATH=${DEPLOY_JAR_PATH}"
+          sudo cp "${BUILD_JAR_PATH}" "${DEPLOY_JAR_PATH}"
+          echo "New JAR file copied."
+
+          # 3. 새로운 WAS 실행
+          echo "Starting new Spring WAS..."
+          LOG_PATH="${DEPLOY_DIRECTORY}/application.log"
+          echo "LOG_PATH=${LOG_PATH}"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)